cli.js

```diff
@@ -342,17 +372,17 @@ TODO COMMANDS
 
 function changeExt(fileName, newExt) {
     let S = fileName;
-    while ((S.length>0) && (S[S.length-1] != ".")) S = S.slice(0, S.length-1);
-    if (S.length>0) {
+    while ((S.length > 0) && (S[S.length - 1] != ".")) S = S.slice(0, S.length - 1);
+    if (S.length > 0) {
         return S + newExt;
     } else {
-        return fileName+"."+newExt;
+        return fileName + "." + newExt;
     }
 }
 
 // r1cs export circomJSON [circuit.r1cs] [circuit.json]
 async function r1csInfo(params, options) {
-    const r1csName = params[0] ||  "circuit.r1cs";
+    const r1csName = params[0] || "circuit.r1cs";
```

Comment:
@xavi-pinsach this is an example of unrelated formatting that you have committed. They are littered throughout the PR. I assume you are using an editor that is formatting on save. Can you please reduce the diff to only the fflonk changes so we don't need to sift through your formatter changes?